src/en/superstream/src/eu/kanade/tachiyomi/animeextension/en/superstream/SuperStream.kt

```diff
@@ -27,7 +27,7 @@ class SuperStream : ConfigurableAnimeSource, AnimeHttpSource() {
 
     override val name = "SuperStream"
 
-    override val baseUrl by lazy { preferences.getString("preferred_domain", superStreamAPI.apiUrl)!! }
+    override val baseUrl = "https://i1.sndcdn.com/artworks-000570872354-fmnvll-t500x500.jpg"
```

Comment:
Can I ask why?
```suggestion
    override val baseUrl = superStreamAPI.apiUrl
```

Reply:
Assumed the joke is better that showing an error in WebView